xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -706,7 +706,10 @@ void CGUIWindowMusicBase::UpdateButtons()
   msg2.SetLabel(g_localizeStrings.Get(744)); // Files
   g_windowManager.SendMessage(msg2);
 
-  msg2.SetLabel(g_localizeStrings.Get(15100)); // Library
+  msg2.SetLabel(g_localizeStrings.Get(14022)); // Library
```

Comment:
Any reason you changed the label here? Both say "Library" but one is used here and one in settings.